README.md

```diff
@@ -1,35 +1,35 @@
-# Support & Care for Maven
-This repository is used to track the work of [Support & Care for Maven](https://open-elements.com/support-care-maven/) as open and transparent as possible.
+# Support & Care for Apache Maven
+This repository is used to track the work of [Support & Care for Apache Maven](https://open-elements.com/support-care-maven/) as open and transparent as possible.
 
 <img width="1720" alt="logo" src="https://github.com/OpenElements/maven-support-care/assets/9443847/875e83bf-26e1-4488-a1fb-44e5de639f78">
 
-The **Support & Care for Maven** Initiative is a groundbreaking program to secure the future of the Maven project through sustainable support and care.
-The initiative aims to strengthen the development and maintenance of Maven through funding, thus ensuring its high quality and availability that the Java ecosystem needs.
+The **Support & Care for Maven** Initiative is a groundbreaking program to help secure the future of the [Apache Maven project](https://maven.apache.org/) through sustainable support and care.
+The initiative aims to strengthen the development and maintenance of Maven through funding of individuals involved in the project, thus ensuring its high quality and availability that the Java ecosystem needs.
```

Comment:
@hboutemy I'm afraid I have to disagree with this change. Our goal is not to create funding for individuals. Our goal is to create / get funding for work done on the project. My suggestion:

```suggestion
The initiative aims to strengthen the development and maintenance of Maven through funding of needed contributions to the project, thus ensuring the high quality and availability of Apache Maven that the Java ecosystem needs.
```

Reply:
@hendrikebbers I love it, thank you